src/lib.rs

```diff
@@ -74,6 +86,16 @@ const PING_TIMEOUT: Duration = Duration::from_millis(2000);
 /// How often to re-broadcast our Join message if we don't know about any other peers right now.
 const REBROADCAST_INTERVAL: Duration = Duration::from_millis(10000);
 
+/// How many characters from the full mesh fingerprint to use in short fingerprints (which are sent
+/// as part of some SwimMessage values).
+const SHORT_FINGERPRINT_LENGTH: usize = 8;
```

Comment:
SHA-256 hashes are 64 hexadecimal digits long when represented as strings, which is a heck-load of overhead to add to our ping acks. A substring is sufficient for our "hey do we agree on the state of the world? no?" sniff test.

Reply:
We talked about this in Slack and realized that since we're just trying to detect that two peers don't exactly agree, the humble CRC32 is more than sufficient—especially since we don't have to worry about malicious tampering here. CRC32s are incredibly efficient (the library we're using uses SIMD instructions on x64, and aarch64 even has a native crc32 instruction built in!) and only take up 4 bytes, which is perfect for something that's going to be included in every ack we send.